atmo/clusters/views.py

```diff
@@ -25,10 +31,10 @@ def new_cluster(request):
             )
         )
         return redirect('keys-new')
-    initial = {
-        'identifier': '{}-telemetry-analysis'.format(user_display(request.user)),
-        'size': 1,
-    }
+    elif ssh_key_count == 1:
+        # If only 1 ssh key, make it pre-selected.
+        initial['ssh_key'] = request.user.created_sshkeys.values('pk')[0]['pk']
```

Comment:
If you're ok with defaulting to an arbitrary ssh key I could do that. I was using `values` so the SQL is more "direct" and to avoid creating a `SSHKey` class just to pluck out the `pk`.